lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -263,26 +264,13 @@ public static function customerListFunction(array $params, $template)
             if (!empty($params['selecttip'])) {
                 $result .= ' ' . self::tipFunction(array('text' => $params['selecttip']), $template);
             } else {
-                $result .= ' ' . self::tipFunction(array('text' => 'Select customer (optional)'), $template);
+                $result .= ' ' . self::tipFunction(array('text' => $firstoption . (empty($required) ? null : '(optional)')), $template);
             }
 
             $result .= '">' . PHP_EOL;
 
-            if (isset($params['firstoption'])) {
-                if (!empty($params['firstoption'])) {
-                    $result .= '<option value="0"';
-                    if (empty($params['selected'])) {
-                        $result .= ' selected';
-                    }
-                    $result .= '>' . trans($params['firstoption']) . '</option>';
-                }
-            } else {
-                $result .= '<option value="0"';
-                if (empty($params['selected'])) {
-                    $result .= ' selected';
-                }
-                $result .= '>' . trans("— select customer —") . '</option>';
-            }
+            $result .= '<option value="0"' . (empty($params['selected']) ? ' selected' : null) . '>' . $firstoption . '</option>';
```

Comment:
Tutaj to samo.

Reply:
https://github.com/chilek/lms/pull/2264/commits/3455e0b4153567e868a1b10faf86d95bb591ef86